server/ladder_service.py

```diff
@@ -495,19 +497,23 @@ def make_game_options(player: Player) -> GameLaunchOptions:
                         )
             await game.wait_launched(60 + 10 * len(all_guests))
             self._logger.debug("Ladder game launched successfully %s", game)
+            metrics.matches.labels(queue.name, MatchLaunch.SUCCESSFUL).inc()
```

Comment:
Does this work actually? The types are a bit weird here since `MatchLaunched.SUCCESSFUL` is an Enum not a string. If you did `MatchLaunch.SUCCESSFUL.value` it would definitely work, but maybe the library knows how to do that automatically?